helm-chart/README.md

```diff
@@ -40,6 +42,11 @@ Override only specific fields of the `values.yaml` file, with the `--set` option
 helm install -n mdcs-test --create-namespace mdcs-helm-test . --set cdcs.imagePullPolicy=IfNotPresent
 ```
 
+The CDCS container can be configured using environment variables. 
+The complete list of environment variables can be found in the
+[customized deployment](https://github.com/usnistgov/cdcs-docker?tab=readme-ov-file#1-customize-the-deployment)
+section of the CDCS Docker documentation.
+
```

Comment:
Can we add more info on how to create these variables in this case and make sure they are used by the app?